### DISCUSSION TITLE
New libraries - matplotlib and schemdraw

### DISCUSSION THREAD

**Discussion by eddygrinder** [micropython/micropython#12814] [ESP32]
Hello,
Is it possible (or anyone have already installed) these two libraries matplotlib and schemdraw on ESP32?

Thank you

**Comment by rkompass** [comment]
Certainly not.
Matplotlib is a monster regarding options/configurability and code size.
I started to work on a minimal graphics library in Javascript that would allow to draw line plots in a matplolib-like manner through a web server.
